README.md

```diff
@@ -259,6 +259,36 @@ h Name=refs/heads/master Repo=terraform-aws-eks
 
 ```
 
+## Rate Limiting
+
+git-xargs attempts to be a good citizen as regards consumption of the GitHub API. git-xargs conforms to GitHub's API [integration guidelines](https://docs.github.com/en/rest/guides/best-practices-for-integrators#dealing-with-secondary-rate-limits).
+
+In addition, git-xargs includes several features and flags to help you: 
+
+1. run jobs without tripping GitHub's rate limits
+1. recover when rate limited by automatically re-trying failed pull requests again, still while honoring the GitHub rate limits
+
+**Distinct processing channels for expensive and non-expensive work**
+git-xargs performs all work up to issuing the API call to open a pull request in parallel - according to the concurrency limits the user might optionally specify by way of the `--max-concurrent-repos` flag.
```

Comment:
```suggestion
git-xargs distinguishes between work that is safe to perform in parallel, such as certain git operations, and work that should be done with consideration and care, such as issuing open pull requests to GitHub's API. 

Therefore, git-xargs is able to perform all safe work as quickly as possible by leveraging goroutines, while treating the more expensive open pull request API calls separately. 

 The `--max-concurrent-repos` flag allows you to specify the upper limit of concurrent goroutines that should be allowed to carry out concurrency-safe work. 
 
Pull requests are also handled on a separate channel so that they can be buffered and retried in accordance with what rate limiting feedback git-xargs is receiving from GitHub's API.  
```